vendor/ansible/roles/contiv_network/tasks/ovs.yml

```diff
@@ -11,7 +15,7 @@
         url: "https://cisco.box.com/shared/static/zzmpe1zesdpf270k9pml40rlm4o8fs56.rpm",
```

Comment:
I thought this was not needed anymore, since we can use some public repo for this. @neelimamukiri ?